src/app/clusters/door-lock-server/door-lock-server.h

```diff
@@ -78,6 +78,14 @@ static constexpr size_t DOOR_LOCK_MAX_USER_NAME_SIZE = 10; /**< Maximum size of
 static constexpr size_t DOOR_LOCK_USER_NAME_BUFFER_SIZE =
     DOOR_LOCK_MAX_USER_NAME_SIZE + 1; /**< Maximum size of the user name string (in bytes). */
 
+enum class AttributeNullabilityType : uint8_t
```

Comment:
Why is this public?  This is only used in a private method of DoorLockServer; it should be declared in the private section of DoorLockServer.

Reply:
done.